mlflow/store/artifact/azure_blob_artifact_repo.py

```diff
@@ -93,7 +93,7 @@ def list_artifacts(self, path=None):
         if path:
             dest_path = posixpath.join(dest_path, path)
         infos = []
-        prefix = dest_path + "/"
+        prefix = dest_path if dest_path and dest_path[-1] == "/" else dest_path + "/"
```

Comment:
```suggestion
        prefix = dest_path if dest_path.endswith("/") else dest_path + "/"
```

Can we use "endswith" here?

Reply:
Thanks @harupy , I have made that change :D